client/src/pages/ChatRoom/ChatArea/ChatDisplay/ChatDisplay.tsx

```diff
@@ -2,35 +2,32 @@ import React from "react";
 
 import { Message } from "./Message/Message";
 
-import { Message as MessageType } from "../../../../types/Message";
-import { User } from "../../../../types/User";
+import { Message as MessageType } from "types/Message";
+import { User } from "types/User";
 
 import "./chat-display.css";
-import { DEFAULT_AVATAR } from "../../../../constants";
+import { DEFAULT_USER } from "./../../../../constants";
```

Comment:
Again long relative paths? Use absolute paths

Reply:
the absolute path isn't working with constants.ts file